downstream/titles/eda/eda-getting-started-guide/master.adoc

```diff
@@ -9,7 +9,7 @@ include::attributes/attributes.adoc[]
 
 
 // Book Title
-= Getting Started with Event-Driven Ansible Guide
+= Getting started with event-driven Ansible guide
```

Comment:
Event-Driven Ansible is a proper noun and should be capitalized

Getting Started with Event-Driven Ansible guide